### PR TITLE
API-3251: Updating the claim establisher job to pull and trigger flashes

### PR DESCRIPTION
**Pull request by SlakrHakr**
## Description of change
Publish any flashes submitted with a 526 on successful claim establishment.

## Original issue(s)
https://vajira.max.gov/browse/API-3251

## Things to know about this PR
Tested hitting endpoint to submit 526 claim locally.
Updated specs.

### CODE REVIEW

File: modules/claims_api/app/workers/claims_api/claim_establisher.rb [hunk 1]
```diff
@@ -20,6 +21,9 @@ def perform(auto_claim_id)
       auto_claim.evss_id = response.claim_id
       auto_claim.status = ClaimsApi::AutoEstablishedClaim::ESTABLISHED
       auto_claim.save
+
+      flashes = JSON.parse(form_data).dig('form526', 'veteran', 'flashes')
```
Comment (bastosmichael) [inline modules/claims_api/app/workers/claims_api/claim_establisher.rb:25]: We may need to eventually pull this from the model but this is okay for now.

**Review by bastosmichael — APPROVED** [review]
LGTM, Keep in mind that these evss_headers are used to send payloads to EVSS, let's make sure that we won't break our EVSS payload submission if we add headers that EVSS isn't expecting meant for BGS. Might want to look at whether or not we are going to break the EVSS stuff by doing this?

**Comment by SlakrHakr** [comment]
> LGTM, Keep in mind that these evss_headers are used to send payloads to EVSS, let's make sure that we won't break our EVSS payload submission if we add headers that EVSS isn't expecting meant for BGS. Might want to look at whether or not we are going to break the EVSS stuff by doing this?

I poked around at where these headers are used for EVSS and every place i tracked down had each header key specifically used, which made me feel good about tacking on headers. The fact that these headers already had a few headers tacked on to the base list (even if they were also EVSS) made me feel pretty solid about tacking my own BGS ones on there as well. I'll doublecheck a few things and comment back here before merging this in though :+1:

**Comment by SlakrHakr** [comment]
Ok, doublechecked and I don't see anything that should be inadvertently affected from the addition to the auth_headers hash. Going to go forward with the merge.

**Review by kfrz — APPROVED** [review]
This looks good to me.

**Pull request merged**